components/collector/src/source_collectors/sonarqube/security_warnings.py

```diff
@@ -45,17 +45,12 @@ async def _get_source_responses(self, *urls: URL, **kwargs) -> SourceResponses:
                 )
             )
         if "security_hotspot" in security_types:
-            # Note: SonarQube is a bit inconsistent. For issue search, the SonarQube status parameter is called
-            # "statuses", but for hotspots it's called "status".
```

Comment:
Shouldn't we leave this comment in somewhere, so we don't mistakenly fix it again? Or is this new API a drop-in replacement?

Reply:
It's tempting, but since we don't use the status parameter of the hotspot search endpoint anymore it feels a bit weird to keep a comment about it in the code.